gpt_index/indices/vector_store/vector_indices.py

```diff
@@ -367,3 +371,53 @@ def __init__(
             vector_store=vector_store,
             **kwargs,
         )
+
+
+class GPTOpensearchIndex(GPTVectorStoreIndex):
```

Comment:
can we add this to `gpt_index/indices/vector_store/__init__.py`? Then users can do `from llama_index.indices.vector_store import GPTOpensearchIndex` (including in the demo notebooks)